omop_file_validator.py

```diff
@@ -118,10 +118,8 @@ def find_error_in_file(column_name, cdm_column_type, submission_column_type,
 
         try:
             if i <= len(df) - 1:
-                if row[column_name]:
```

Comment:
NaN values were being accepted by this condition and compared via cast_type, which would often fail.

Reply:
Thanks!